### ISSUE TITLE
Csharp implementation of the official Camunda 8 Connector SDK

### ISSUE THREAD

**Issue by vincentgiraud** [camunda/connector-sdk#294]
__Is your feature request related to a problem? Please describe.__

An implementation of Camunda 8 connector SDK in Csharp allowing the writing of connectors for .NET

__Describe the solution you'd like__

 A port of java official https://github.com/camunda/connector-sdk 

__Describe alternatives you've considered__

I'm using https://github.com/camunda-community-hub/zeebe-client-csharp but I can't use modeler templates with dtos validation and secret management

__Additional context__

This is a request from a Fortune 500 customer :)

**Comment by MaxTru** [comment]
Makes sense. But this is not a priority at the moment. Moving to backlog, we will tackle this someday, maybe. Also happy to retrieve contributions.
